packages/twenty-front/src/modules/activities/ask-ai/right-drawer/components/SQLQueryBuilder.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentProps } from 'react';
```

Comment:
Let's merge main into your branch, it's getting old and you'll see it'll feel much nicer as I moved the commandbar to the right side :)